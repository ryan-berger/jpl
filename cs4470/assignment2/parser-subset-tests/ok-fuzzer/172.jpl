time return 8.
print ";"
time print ";/"
show C()
return .2
print ""
