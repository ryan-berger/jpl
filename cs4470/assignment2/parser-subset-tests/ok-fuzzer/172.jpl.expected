(TimeCmd (StmtCmd (ReturnStmt (FloatExpr 8))))
(PrintCmd ";")
(TimeCmd (PrintCmd ";/"))
(ShowCmd (CallExpr C))
(StmtCmd (ReturnStmt (FloatExpr 0)))
(PrintCmd "")
